.github/workflows/docker-publish.yml

```diff
@@ -33,14 +33,11 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      # Install the cosign tool except on PR
```

Comment:
This part (installation of cosign) can be removed, as we don't use cosign anymore.